std/concurrency.d

```diff
@@ -656,6 +634,36 @@ body
     thisInfo.ident.mbox.get( ops );
 }
 
+///
+@system unittest
+{
+    import std.variant;
+    __gshared string received;
```

Comment:
The documentation examples are supposed to show the best practices of using the provided functionality, in the specific case - means to avoid memory sharing and low-level synchronization by using the message passing paradigm. Please refrain from using global variables in the examples of this module, especially `__gshared` ones.

The examples should ideally look as if you could easily scale from threads, to process and to network nodes, i.e avoid sharing of state.